src/actions/DeleteWalletModalActions.js

```diff
@@ -18,24 +13,19 @@ export const showDeleteWalletModal = (walletId: string, additionalMsg: string =
   const { account } = state.core
   const { currencyWallets = {} } = account
 
-  // Use `launchModal` to put the modal component on screen:
-  const modal = createYesNoModal({
-    title: s.strings.fragment_wallets_delete_wallet,
-    message: (
-      <Text style={{ textAlign: 'center' }}>
-        {s.strings.fragmet_wallets_delete_wallet_first_confirm_message_mobile}
-        <B>{`${getWalletName(currencyWallets[walletId])}?`}</B>
-        {`\n \n ${additionalMsg}`}
-      </Text>
-    ),
-    icon: <OptionIcon iconName={DELETE} />,
-    noButtonText: s.strings.string_cancel_cap,
-    yesButtonText: s.strings.string_delete
-  })
+  const resolveValue = await Airship.show(bridge => (
+    <ButtonsModal
+      bridge={bridge}
+      title={s.strings.fragment_wallets_delete_wallet}
+      message={`${s.strings.fragmet_wallets_delete_wallet_first_confirm_message_mobile} ${getWalletName(currencyWallets[walletId])}?\n\n${additionalMsg}`}
```

Comment:
Maybe we should fix `ButtonsModal` to take a `React.Node | string` for the message, so we can still do the bold text.